packages/app/src/specs/SpecsListRow.vue

```diff
@@ -27,7 +27,10 @@
           class="min-w-16px max-w-16px min-h-16px icon-dark-gray-300 group-hocus:icon-dark-indigo-500 max-h-16px"
         />
       </span>
-      <div :class="classes?.gitText">
+      <div
+        class="overflow-hidden truncate"
```

Comment:
I'm not so sure about truncating. If we don't want to make the rows taller & wrap the text, maybe we could allow horizontal scroll. Mainly I'm thinking if the data is useful at full width, it's still useful if somebody is choosing to make the window smaller - or triggers the truncate through browser zoom controls.

Reply:
How about `:title="specName"`? You could just hover it with your mouse.

Reply:
I resolved the conflicts with the main branch and added `:title` along the way. Happy to discuss the UI further - since the correct behavior isn't obvious from the mocks, I'm just going to merge this one up since I don't see any obvious problems.

Let's discuss how this should behave, once we have a consensus on the correct behavior, we can make another PR to amend it.